plexpy/plexivity_import.py

```diff
@@ -32,8 +32,9 @@ def extract_plexivity_xml(xml=None):
     clean_xml = helpers.latinToAscii(xml)
     try:
         xml_parse = minidom.parseString(clean_xml)
-    except:
+    except Exception as e:
         logger.warn(u"PlexPy Importer :: Error parsing XML for Plexivity database.")
+        logger.exception(e)
```

Comment:
For example these can be changed to:
```
logger.error(u"PlexPy Importer :: Error parsing XML for Plexivity database: %s" % e)
```

Instead of having two logger messages.